### ISSUE TITLE
Empty SPS for MSE H265

### ISSUE THREAD

**Issue by AlexxIT** [AlexxIT/go2rtc#99]
```
17:02:04.189 WRN [api.mse] init error="empty SPS: &streamer.Codec{Name:"H265", ClockRate:0x15f90, Channels:0x0, FmtpLine:"", PayloadType:0x60}"
```

**Comment by banthungprong** [comment]
Thanks! I didn't connect this error in the frigate logs with go2rtc.
I am using HikVision Cameras with H265 or H265+.

**Comment by AlexxIT** [comment]
Should be fixed in `v0.1-rc.4`
